### ISSUE TITLE
Remove Snapshot version from pom 

### ISSUE THREAD

**Issue by martinezmatias** [SpoonLabs/gumtree-spoon-ast-diff#116]
The pom references to a snapshot version of Spoon:
https://github.com/SpoonLabs/gumtree-spoon-ast-diff/blob/master/pom.xml#L41

**Comment by monperrus** [comment]
Yes, we need to release 6.8 and change from SNAPSHOT to release, see
https://github.com/INRIA/spoon/pull/3163

**Comment by aaronemassey** [comment]
Same with for POM of 1.21

```
<!-- must be set to 7.6.0 before releasing a new version --> 
      <version>7.6.0-SNAPSHOT</version>

```

**Comment by monperrus** [comment]
closed per  #117

**Comment by monperrus** [comment]
Thanks for the bug report and reminder
